socialNetwork/docker-compose-swarm.yml.template

```diff
@@ -495,6 +495,13 @@ services:
     deploy:
       placement:
         constraints: [node.hostname == node-3.<<EXPERIMENT_NAME>>.infosphere-pg0.<<SSH_HOST>>]
+
+  io-intensive:
+    image: 127.0.0.1:5000/io_intensive
+    deploy:
+      placement:
+        constraints: [node.hostname == node-3.<<EXPERIMENT_NAME>>.infosphere-pg0.<<SSH_HOST>>]
```

Comment:
Do you plan to allow users to configure on which node these resource exploiting containers should run?